tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Debug/ProjectLaunchTargetsProviderTests.cs

```diff
@@ -519,11 +515,9 @@ public async Task QueryDebugTargetsAsync_NonConsoleAppLaunch_DoesNotIncludeInteg
         }
 
         [Theory]
-        [InlineData("winexe")]
-        [InlineData("appcontainerexe")]
-        [InlineData("library")]
-        [InlineData("WinMDObj")]
```

Comment:
What happened to these other project output types?

Reply:
As far as the `ProjectLaunchTargetsProvider` is concerned, there are effectively only three kinds of projects: libraries, console apps, and everything else. It doesn't actually care about the project's output type; that's an implementation detail of `OutputTypeChecker`. Since `ProjectLaunchTargetsProvider` no longer needs to care about _how_ `IOutputTypeChecker` is implemented, we can remove that detail from the tests.